WcaOnRails/app/webpacker/components/Panel/Wfc/index.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  Dropdown,
+  Grid, Header, Menu, Segment,
+} from 'semantic-ui-react';
+import {
+  panelWfcPageDataUrl,
+  countryBandsUrl,
+} from '../../../lib/requests/routes.js.erb';
+import useLoadedData from '../../../lib/hooks/useLoadedData';
+import Errored from '../../Requests/Errored';
+import Loading from '../../Requests/Loading';
+import DelegateProbations from '../../DelegateProbations';
+import DuesExport from './DuesExport';
+
+const sections = [
+  {
+    id: 'dues-export',
+    name: 'Dues Export',
+    Component: DuesExport,
+  },
+  {
+    id: 'country-bands',
+    name: 'Country Bands',
+  },
+  {
+    id: 'delegate-probations',
+    name: 'Delegate Probations',
+    Component: DelegateProbations,
+    forAtleastSeniorMember: true,
+  },
+];
+
+export default function Wfc() {
+  const { data, loading, error } = useLoadedData(panelWfcPageDataUrl);
+  const [selectedMenu, setSelectedMenu] = React.useState(sections[0].id);
+  const SelectedComponent = React.useMemo(() => sections.find(
+    (section) => section.id === selectedMenu,
+  ).Component, [selectedMenu]);
+
+  if (loading) return <Loading />;
+  if (error) return <Errored />;
+
+  function menuClickHandler(value) {
+    if (value === 'country-bands') {
+      window.open(countryBandsUrl, '_blank');
+      return;
+    }
+    setSelectedMenu(value);
+  }
```

Comment:
Even though currently country band opens in new tab, plan is to migrate that as well to react in later stage, so that it will come within the panel itself. Also, the panel doesn't have any home page or anything, it will just have three items in menu. So I guess it's fine to have it as menu thinking that the react migration of country_bands will come soon?

And ya I've posted the screenshot in description.

Reply:
Thanks for the screenshots, very helpful!

I have 2 issues with the country bands being a button in disguise in the menu (aka section/tab list). I think it should just be a separate button somewhere (underneath the menu?).

1. It's unintuitive as a user. All the identical-looking elements in a group should behave identically.
2. It complicates the logic of the menu and makes it harder to understand the code. And if the country band is going to later become a tab anyway, then this logic will just need to be pulled out later - might as well keep it simple from the start.

Point 2 is only partially relevant since you already have the changes in there, but keeping the code simpler in the meantime still applies, as well as point 1.

On the other hand, if it's removed then in some cases there will only be one tab in the menu, which might be odd. But we could just hide the menu in that case.

But I'd be curious to hear what other people think.

Reply:
Makes sense to me. But I don't know having separate link below menu looks good or not. Anyway, I've added an 'open in new tab' icon and updated the description. Can you please check if the current one is fine as it is no longer 'identical elements'?

Reply:
Yeah that mostly resolves my concerns, thanks.